lib/client/src/client.rs

```diff
@@ -666,37 +666,40 @@ impl MangoClient {
     pub fn serum3_close_open_orders_instruction(
         &self,
         market_index: Serum3MarketIndex,
-    ) -> Instruction {
+    ) -> PreparedInstructions {
         let account_pubkey = self.mango_account_address;
         let s3 = self.context.serum3(market_index);
 
         let open_orders = self.serum3_create_open_orders_address(market_index);
 
-        Instruction {
-            program_id: mango_v4::id(),
-            accounts: anchor_lang::ToAccountMetas::to_account_metas(
-                &mango_v4::accounts::Serum3CloseOpenOrders {
-                    group: self.group(),
-                    account: account_pubkey,
-                    serum_market: s3.address,
-                    serum_program: s3.serum_program,
-                    serum_market_external: s3.serum_market_external,
-                    open_orders,
-                    owner: self.owner(),
-                    sol_destination: self.owner(),
-                },
-                None,
-            ),
-            data: anchor_lang::InstructionData::data(
-                &mango_v4::instruction::Serum3CloseOpenOrders {},
-            ),
-        }
+        PreparedInstructions::from_single(
+            Instruction {
+                program_id: mango_v4::id(),
+                accounts: anchor_lang::ToAccountMetas::to_account_metas(
+                    &mango_v4::accounts::Serum3CloseOpenOrders {
+                        group: self.group(),
+                        account: account_pubkey,
+                        serum_market: s3.address,
+                        serum_program: s3.serum_program,
+                        serum_market_external: s3.serum_market_external,
+                        open_orders,
+                        owner: self.owner(),
+                        sol_destination: self.owner(),
+                    },
+                    None,
+                ),
+                data: anchor_lang::InstructionData::data(
+                    &mango_v4::instruction::Serum3CloseOpenOrders {},
+                ),
+            },
+            self.context.compute_estimates.health_cu_per_serum,
```

Comment:
now that I think about it, health_cu_per_serum isn't the right choice. Use cu_per_mango_instruction - it's a large overestimate but 6k cu is too little